### PR TITLE
Distribution Gallery: Beta Alternative Parametrization

### PR DESCRIPTION
**Pull request by aleicazatti**

### CODE REVIEW

**Comment by OriolAbril** [comment]
The logs are complaining about unknown directive jupyter-execute. Is it installed and "activated" in conf.py?

**Comment by OriolAbril** [comment]
Library that provides the directive: https://jupyter-sphinx.readthedocs.io/en/latest/

**Comment by aleicazatti** [comment]
> The logs are complaining about unknown directive jupyter-execute. Is it installed and "activated" in conf.py?

This was the problem. Now it works. Thank you very much!

File: docs/examples/gallery/beta.md [hunk 1]
```diff
@@ -33,20 +46,81 @@ for alpha, beta in zip(alphas, betas):
     ax = Beta(alpha, beta).plot_pdf()
 ax.set_ylim(0, 5);
 ```
+:::::
+
+:::::{tab-item} Parameters $\mu$ and $\sigma$  
+```{jupyter-execute}
+:hide-code:
+
+from preliz import Beta, style
```
[the remark this answers is not in the document]
Comment (OriolAbril) [reply docs/examples/gallery/beta.md:55]: Calls to `jupyter-execute` within the same file are like cells in a notebook, the results of previous ones are stored and available

**Comment by aloctavodia** [comment]
The PDF tabs and the CDFs tabs should be synced, so changing one, changes the other. Or we should have a single plot with two subplots on top if each other. One for the PDF and another for the CDF. If we have the two subplots the titles should indicate if the plot is the PDF or CDF.

**Pull request merged**